### PR TITLE
Removing the non-derived field that can be queried using filters

### PR DESCRIPTION
**Pull request by this-username-is-taken**

### CODE REVIEW

File: schema-lending-extended.graphql [hunk 1]
```diff
@@ -1001,11 +969,25 @@ type Position @entity {
   dailySnapshots: [PositionSnapshot!]! @derivedFrom(field: "position")
 }
 
+# Unlike other snapshots that are taken at a fixed time interval. Position
+# snapshots should be taken after every event, including the opening and
+# closing events. This will prevent an ever growing number of snapshots
+# for positions that are not moving. As we are only recording the balance
+# in token amounts instead of in USD, this will work well.
 # Note that we only take snapshot for open positions
 type PositionSnapshot @entity {
```
Comment (0xbe1) [inline schema-lending-extended.graphql:978]: the entity doesn't come with a timestamp field, then how do we order them?

Comment (this-username-is-taken) [reply schema-lending-extended.graphql:978]: which entity doesn't come with a timestamp field?

File: schema-lending-extended.graphql [hunk 2]
```diff
@@ -1001,11 +969,25 @@ type Position @entity {
   dailySnapshots: [PositionSnapshot!]! @derivedFrom(field: "position")
```
[the remark this answers is not in the document]
Comment (this-username-is-taken) [reply schema-lending-extended.graphql:969]: ah, nice catch!

File: schema-lending-extended.graphql (hunk 1 above)
Comment (0xbe1) [reply schema-lending-extended.graphql:978]: my bad, saw it

**Pull request merged**